catalog/dags/common/loader/provider_details.py

```diff
@@ -124,7 +124,8 @@
 
 # User-Agent header for APIs that require it
 CONTACT_EMAIL = os.getenv("CONTACT_EMAIL")
-UA_STRING = f"Openverse/0.1 (https://wordpress.org/openverse; {CONTACT_EMAIL})"
+CANONICAL_ORIGIN = os.getenv("CANONICAL_ORIGIN", "https://openverse.org")
```

Comment:
I guess if we want to match the API's new variables, it should be an environment variable of `CANONICAL_DOMAIN` with origin derived from it, but I don't know if that was a goal with this PR. It doesn't matter to me either way, as there's so little overlap between the variables for each I don't think we need to be concerned with having them match _except_ when the names are this similar, it would be easier to make a quick mistake.

To clarify, not a suggestion for a change, just noting the slight difference, will at least help me keep it in mind during infra work to migrate Airflow's environment variables to the new approach.

Reply:
Yes, I intended to make the change aligned with your on moving the domain :) I thought the `CANONICAL_DOMAIN` was a variable we could skip here, but you're right. It would be better to match the API's variables in this case.

Done ✅